scripts/package

```diff
@@ -5,21 +5,53 @@ source $(dirname $0)/version
 
 cd $(dirname $0)/..
 
-PROJECT=`basename "$PWD"`
+project=$(basename "$PWD")
 
-if [ ! -x ./bin/longhorn ]; then
-    ./scripts/build
+command -v buildx >/dev/null && build_cmd=(buildx) || build_cmd=(docker buildx)
+
+# read configurable parameters
+REPO=${REPO:-longhornio}
+IMAGE_NAME=${IMAGE_NAME:-$project}
+TAG=${TAG:-''}
+OUTPUT_ARGS=${OUTPUT_ARGS:-'--load'}
+IS_SECURE=${IS_SECURE:-'false'}
+MACHINE=${MACHINE:-''}
+TARGET_PLATFORMS=${TARGET_PLATFORMS:-''}
+IID_FILE=${IID_FILE:-''}
+IID_FILE_FLAG=${IID_FILE_FLAG:-''}
```

Comment:
No used for community image build, correct?

Reply:
Yeah, not needed for community build. It is used in secure build only.